shared/teams/index.js

```diff
@@ -3,7 +3,11 @@ import Render from './render'
 import pausableConnect from '../util/pausable-connect'
 import openURL from '../util/open-url'
 
-const mapStateToProps = state => ({})
+import type {TypedState} from '../constants/reducer'
+
+type StateProps = {}
+
+const mapStateToProps = (state: TypedState): StateProps => ({})
```

Comment:
if you don't actually map any state to props you can just pass null to connect

Reply:
Done.